doc/osh.1

```diff
@@ -37,37 +37,25 @@ flags, with the following differences:
 .It Fl n
 Only validate the syntax.
 Also prints the AST.
-.It Fl Fl show-ast
+.It Fl -show-ast
 Print the AST in addition to executing.
-.It Fl Fl ast-format
-What format the AST should be in
+.It Fl -ast-format
+What format the AST should be in.
 .El
 .Sh EXIT STATUS
-.Bl -dash
-.It
-1 for runtime errors.
-.Pp
-Examples:
-.Ql echo foo > out.txt
-and the file cannot be opened.
-.It
-2 for parse errors.
-This means that we didn't even attempt to do anything, rather than doing something and it failing.
-.Bl -bullet
-.It
-shell language parse error.
-.It
-builtin usage error, e.g.
-.Ql read -Z
-.El
-.It
-POSIX mentions these:
-.Bl -bullet
-.It
-126 for permission denied when running a command (errno EACCES)
-.It
-127 for command not found
-.El
+The
+.Nm
+utility exits with one of the following values
```

Comment:
I forgot to put a colon at the end of `values`, whoops

Reply:
done